apps/backend/src/modules/contributors/contributor.repository.ts

```diff
@@ -73,6 +73,11 @@ class ContributorRepository implements Repository {
 					"COALESCE(ARRAY_AGG(DISTINCT jsonb_build_object('id', projects.id, 'name', projects.name)) FILTER (WHERE projects.id IS NOT NULL), '{}') AS projects",
 				),
 			)
+			.select(
+				raw(
+					"MAX(CASE WHEN activity_logs.commits_number > 0 THEN activity_logs.date ELSE NULL END) AS last_activity_date",
```

Comment:
Why need this check?
If I'm not wrong, if there is no commits then date is null by default

Reply:
Activity log has commits number field. If last activity has 0 commits then we are looking on the one before it, and so on. There is a requirement that the label should show last activity log with more than zero commits.

Reply:
I got it. I just thought that if `activity_logs.commits_number == 0` then `activity_logs.date` should be `null`. Isn't this the case? If this is the case, then can't we simplify query as 
```suggestion
					"MAX(activity_logs.date) AS last_activity_date",
```
If it isn't the case, then all is good

Reply:
Oh, alright. Maybe I'm missing something out, but as I see, in activity logs repo and service, there is no such check or anything, the date can be present even if there are 0 commits.

Reply:
yes, let's keep this check